crowsnest/v4l2/ctl.py

```
@@ -0,0 +1,250 @@
+    return controls
+
+
+def parse_qc_of_path(device_path: str, qc: raw.v4l2_query_ext_ctrl) -> dict:
+    """
+    Parses the query control to an easy to use dictionary
+    """
+    try:
+        fd = os.open(device_path, os.O_RDWR)
+        controls = parse_qc(fd, qc)
+        os.close(fd)
+        return controls
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)